bindings/python/setup.py

```diff
@@ -112,6 +112,9 @@ def target_specific():
 		extra_link = flags.parse(ldflags)
 		extra_compile = flags.parse(extra_cmd)
 
+		if 'g++' in os.environ['CXX'] or 'clang' in os.environ['CXX']:
```

Comment:
did you see the m4 scripts I added to the other PR that detects how to configure the compiler for C++11 support? it adds the "-std=c++11" argument directly to the CXX variable, which would break this